subcommands/targets/offline-update.go

```diff
@@ -566,6 +568,7 @@ func doCheckBundle(cmd *cobra.Command, args []string) {
 	fmt.Printf("\tThreshold:\t%d\n", rootMeta.Signed.Roles["targets"].Threshold)
 	numberOfMissingSignatures := rootMeta.Signed.Roles["targets"].Threshold - len(bundleTufMeta.Signatures)
 	if numberOfMissingSignatures > 0 {
-		fmt.Printf("\tMissing:\t%d\n", numberOfMissingSignatures)
+		fmt.Printf("\tMissing:\t%d (the number of required additional signatures;"+
+			" run the `sign` sub-command to sign the bundle)\n", numberOfMissingSignatures)
```

Comment:
This specific change belongs to the previous commit, right?

Reply:
Yes.

Reply:
Fixed.